skills/stats.js

```diff
@@ -26,17 +24,19 @@ const presentStats = (bot, message) => {
         users++;
       }
     });
-    fsSync.syncToLocalFile([{ 'registered': users, 'active': active }], STATS_ACTIVE);
-    bot.replyPrivate(message,
-      {
-        text:
-          `*${users}* registered - *${active}* active\n`
-          + `*${stats.slashCommands}* slashCommandActions\n`
-          + `*${stats.triggers}* triggerActions\n`
-          + `*${stats.convos}* conversationStartedActions\n`,
-        link_names: 1, parse: 'full'
-      }
-    );
+    fsSync.syncToLocalFile([{'registered': users, 'active': active}], STATS_ACTIVE);
+    if (message) {
+      bot.replyPrivate(message,
+        {
+          text:
+            `*${users}* registered - *${active}* active\n`
```

Comment:
You don't need to concatenate template literals. They allow multi-line string by default.